.github/workflows/run-pfcp-agent-unit-test.yml

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2022-present Open Networking Foundation
+name: Unit tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  unit-test-pfcpiface:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.4.0
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.13'
```

Comment:
uhoh, that's the source of the error before. `timer.Reset` was added in 1.15.
Now that it's running in Docker, this is not needed anymore?

Reply:
oh right.. I made a blind copy-paste. Apologies. Will remove that